### PR TITLE
Booth Daemon Arguments: disable foreground for D flag

### PR DESCRIPTION
**Pull request by ChrisKowalczyk**
As discussed in [Issue 61](https://github.com/ClusterLabs/booth/issues/61), this change will disable moving Booth to foreground, when debug output is enabled.
We need this change to allow using debug mode in Booth instances acting as HA resources.

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -1116,10 +1116,12 @@ static int read_arguments(int argc, char **argv)
 					strcat(cp, BOOTH_DEFAULT_CONF_EXT);
 			}
 			break;
+
 		case 'D':
 			debug_level++;
 			enable_stderr = 1;
```
Comment (dmuhamedagic) [inline src/main.c]: enable stderr should go below to the 'S' option.

Comment (ChrisKowalczyk) [inline src/main.c]: well spotted, thanks

**Comment by dmuhamedagic** [comment]
The stderr enable should be removed from the 'D' case, because it doesn't do any good if booth runs as a daemon.

**Comment by ChrisKowalczyk** [comment]
@dmuhamedagic, do you mean to remove 'enable_stderr' from the 'S' case?

**Comment by dmuhamedagic** [comment]
No, just to remove it from the 'D' case. Output to stderr makes
sense only if booth runs in the foreground.

**Comment by ChrisKowalczyk** [comment]
Thanks, you are right. I will remove it.

**Pull request merged**